src/libraries/Microsoft.Extensions.DependencyModel/Directory.Build.props

```diff
@@ -0,0 +1,7 @@
+﻿<Project>
+  <Import Project="..\Directory.Build.props" />
+  <PropertyGroup>
+    <StrongNameKeyId>MicrosoftAspNetCore</StrongNameKeyId>
```

Comment:
I'm not seeing where was this set on the installer but I trust that maybe some common files where setting it?

Reply:
It is set here:

https://github.com/dotnet/runtime/blob/15abeaa93110f0d06b741658ba47e75d7584dd9e/src/installer/managed/CommonManaged.props#L31-L32

Reply:
I think @maryamariyan is adding logic in `Directory.Build.props` when the project name starts with `Microsoft.Extensions` to set the `StrongNameKeyId` to `MicrosoftAspNetCore`

Reply:
@safern - That doesn't appear to be working. I think it is getting overwritten somewhere. On my machine all the other `Microsoft.Extensions.*` assemblies have `PublicKeyToken=31bf3856ad364e35`. `MicrosoftAspNetCore`'s public key token is `adb9793829ddae60`.

Reply:
We recently fixed this.  There was an ordering bug in the props.  Make sure you pull latest master.

Reply:
https://github.com/dotnet/runtime/commit/87e7991b847a1c80654e05d81005f0931a04ab71

Reply:
I believe all should work fine without this file.

Reply:
Ok, I have removed this file now that I sync'd with master.